src/frontend/styles/components/sidebar.pcss

```diff
@@ -73,11 +73,36 @@
     }
   }
 
+  &__search {
```

Comment:
There are some differences between the implementation and Figma

Height should be 34
<img width="360" alt="image" src="https://user-images.githubusercontent.com/3684889/195413842-2c33a4f3-7cbd-4856-a140-a451e876d393.png">

The placeholder should have the same color as icon

<img width="114" alt="image" src="https://user-images.githubusercontent.com/3684889/195413956-eb0feb16-8ef6-41b3-af22-19888dc062e8.png">

Margin should be 20px (for now it is 19px)

<img width="375" alt="image" src="https://user-images.githubusercontent.com/3684889/195414182-7672572d-d5f2-42a7-b527-7cc7a1328e28.png">

Reply:
Font Size should be 14px
<img width="335" alt="image" src="https://user-images.githubusercontent.com/3684889/195414760-d5323cd5-1fd4-45b5-8b98-297e07c337f9.png">

Reply:
done